### PR TITLE
Colors in  R, Python, and JS/HTML/CSS 

### PR DESCRIPTION
**Pull request by tracyhsj**

### CODE REVIEW

**Comment by tracyhsj** [comment]
color_r_python_css

**Comment by zcysxy** [comment]
Hi, what is the README in the resources directory about? Would you mind removing it?

File: color_r_python_css.Rmd [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (jl6298) [inline color_r_python_css.Rmd:1]: I'm not sure if this is ok, but, to be safe, please remove this empty line.

**Review by jl6298 — CHANGES_REQUESTED** [review]

**Review by jl6298 — APPROVED** [review]

**Comment by tracyhsj** [comment]
Hi! I've updated the file and merged branch(not sure if this is the right thing to do), let me know if there is any other problem. Thank you!

**Pull request merged**